src/renderers/SwitchableRenderer.js

```diff
@@ -82,7 +82,7 @@ export default class SwitchableRenderer extends BaseRenderer {
             choices.forEach((choiceRenderer) => {
                 if (choiceRenderer) {
                     const cr = choiceRenderer;
-                    cr.on(RendererEvents.COMPLETE_START_BEHAVIOURS, () => {
+                    cr.on(RendererEvents.COMPLETE_START_BEHAVIOURS, () => { // TODO
```

Comment:
```suggestion
                    cr.on(RendererEvents.COMPLETE_START_BEHAVIOURS, () => {
```
Looks like we still need to remove this event?